biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -2767,6 +2784,20 @@ subroutine hydraulics_bc ( nsites, sites, bc_in, bc_out, dtime)
            end do
 
            ! Second pass, apply normalized weighting factors for fluxes
+
```

Comment:
Good catch for low water content.  If the sum of weight is close to zero, then the layer-specific water uptake is proportion of the root x water content here.  It might be better to  explain more on the new disaggregation

Reply:
I updated the comment a little, should be more accurate. Does this look good @xuchongang ?